test_live_sync.py

```diff
@@ -72,6 +72,21 @@ def test_deletestatus(pvo):
     assert delete_result.text == "OK 200: Deleted Status"
 
 
+def test_addoutput(pvo):
```

Comment:
can you test this after merging?

Reply:
I'll try and set up a test for it with a test system, I don't want to mess up my data 😄